example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js

```diff
@@ -103,11 +105,11 @@ describe('The Ingest Granule failure workflow', () => {
 
   afterAll(async () => {
     // clean up stack state added by test
+    await deleteExecution({ prefix: config.stackName, executionArn: workflowExecution.executionArn });
```

Comment:
Is there a reason you don't include this call in the Promise.all below? 🤔

Reply:
Yeah, that's exactly it. We need to be sure that executions are deleted before collections.